### ISSUE TITLE
val_loss did not improve from inf, while loss decreases nicely

### ISSUE THREAD

**Issue by marlon-shiftone** [keras-team/keras#14584]
I have a problem for which I have tried almost everything: 

I'm training a model with a custom loss measure. I have used it many times before and I'm using it which a generator plugged into the fit(). However, in order to decrease the training time I decided to slide the data frame and to train the model with a slice of it, save the model, and continue training with the rest of it. 

Anyway, in every stage of this process I get: `val_loss did not improve from inf`

Here is my custom loss:
```
def loss(y_true,y_pred):
    y_pred = relu(y_pred)
    z = k.maximum(y_true, y_pred)
    y_pred_negativo = Lambda(lambda x: -x)(y_pred)
    w = k.abs(add([y_true, y_pred_negativo])) 
    if k.sum(z) == 0:
        erro = 0
    elif k.sum(y_true) == 0 and k.sum(z) != 0:
        erro = 100
    elif k.sum(y_true) == 0 and k.sum(z) == 0:
        erro = 0
    else:
    	erro = (k.sum(w)/k.sum(z))*100
    return erro
```

I'm really don't know what is happening since the loss decreases very nicely. I have already removed dropouts, batchnormalization, increased the dense size, with no success. 

Any idea is more than welcomed!

**Comment by bguetarni** [comment]
It's quite hard to tell what is happening during the training without any curves. Can you share the code and the data if possible ?
Or at least the curves that you have. Thanks.

**Comment by sushreebarsa** [comment]
@marlon-shiftone In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thanks!

**Comment by sushreebarsa** [comment]
Closing this issue due to lack of recent activity .Please feel free to reopen if you still have concern.Thanks!
